src/app-router.js

```diff
@@ -27,8 +27,8 @@ import { triggerMethod }       from './trigger-method';
 
 var AppRouter = Backbone.Router.extend({
 
-  constructor: function(options) {
-    this.options = options || {};
+  constructor: function(options = {}) {
```

Comment:
I'm not 100% sure we need the default here at all.. this is just attaching options for `getOption` I assume.. and `getOption` will actually work better if it's undefined (one less check)